### PR TITLE
SOC2: Add asset management and 3rd party risk policies

### PR DESCRIPTION
**Pull request by knolleary**
Initial drafts of policies:

 - Asset Management
 - Third-Party Risk Management

### CODE REVIEW

File: src/handbook/company/security/asset-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:52]: Please link to this doc from the peopleops off boarding section.

File: src/handbook/company/security/asset-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:61]: Sure, but let's document what the concrete steps are that must be taken.

File: src/handbook/company/security/asset-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:68]: All these policies are missing the how of this statement.

File: src/handbook/company/security/third-party-risk-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:22]: As it's capitalised, let's also link to the classification document. There's legal significance to the capital letter

File: src/handbook/company/security/third-party-risk-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:28]: Let's link in this PR

File: src/handbook/company/security/third-party-risk-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:48]: How?

File: src/handbook/company/security/third-party-risk-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:27]: So we cannot merge this before we have such a list.

File: src/handbook/company/security/asset-management.md
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:52]: Done

File: src/handbook/company/security/asset-management.md
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:61]: I've added links to the Access Control policy regarding digital security and an explicit statement on not leaving equipment unattended in public spaces. I think that is sufficient in this context.

File: src/handbook/company/security/asset-management.md
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:70]: Agreed. Would a simple 'submitted via email' be sufficient to clarify? That will get a written record of the exception approval without having to create more process and gets this moving forward.

File: src/handbook/company/security/third-party-risk-management.md
Comment (knolleary) [reply src/handbook/company/security/third-party-risk-management.md]: Done - spreadsheet created with an initial list based on the Saas Spend spreadsheet and added link to the vendor section.

File: src/handbook/company/security/third-party-risk-management.md
Comment (knolleary) [reply src/handbook/company/security/third-party-risk-management.md:26]: We now have an initial list that lets us move forward.

**Comment by knolleary** [comment]
@ZJvandeWeg updated for your comments. Only outstanding question is the 'how' of the exception process. Have proposed  a simple change to clarify. If you're happy with that, I'll update all of the existing policy docs to match as part of this PR.

File: src/handbook/company/security/asset-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:71]:
```suggestion
Requests for an exception to this policy must be submitted via email to the
CEO or CTO for approval.
```

File: src/handbook/company/security/third-party-risk-management.md
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:48]:
```suggestion
## Exceptions

Requests for an exception to this policy must be submitted via email to the
CEO or CTO for approval.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**
